filter_plugins/oo_filters.py

```diff
@@ -377,6 +377,10 @@ def oo_parse_certificate_names(certificates, data_dir, internal_hostnames):
             else:
                 certificate['names'] = []
 
+            # We will always look for named certificates in data_dir regardless of their original path.
+            certificate['certfile'] = os.path.join(data_dir, os.path.basename(certificate['certfile']))
+            certificate['keyfile'] = os.path.join(data_dir, os.path.basename(certificate['keyfile']))
```

Comment:
We should probably have this be a subdirectory of data_dir and not data_dir itself, otherwise they could potentially clobber existing certs.

I'd suggest something like `{{ data_dir }}/named_certificates/{{ cert_cn }}/{{ certfile }}`
That would let us avoid clobbering certs that we generate as well as allow them to have multiple certificates that have the same base name referenced.